src/lib/knex.js

```diff
@@ -1,25 +1,17 @@
 'use strict'
 
-function getKnexConfigType () {
-  if (process.env.LEDGER_DB_ENV) {
-    return process.env.LEDGER_DB_ENV
-  }
-  if (process.env.LEDGER_UNIT_DB_URI) {
-    if (process.env.LEDGER_UNIT_DB_URI.startsWith('mysql:')) {
-      return 'mysql'
-    }
-    if (process.env.LEDGER_UNIT_DB_URI.startsWith('postgres:')) {
-      return 'postgres'
-    }
-  }
-  return 'development'
-}
+const config = require('../services/config')
 
 // This module encapsulates Knex instance, which is used to access database.
 // Environment variable LEDGER_DB_ENV should be set according to deployment type
 // E.g., 'production' or 'staging'.
 // This is used to look up Knex configuration in knexfile.js
-const knexConfig = require('../knexfile')[getKnexConfigType()]
+const knexConfigEnv = config.getIn(['db', 'knex_env'])
```

Comment:
Nao decided to rename LEDGER_DB_ENV to LEDGER_DB_KNEX_ENV. I'll update the comment and readme.